src/OpenTelemetry/Metrics/MetricReaderExportType.cs

```diff
@@ -0,0 +1,24 @@
+// <copyright file="MetricReaderExportType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    public enum MetricReaderExportType
+    {
+        Simple,
```

Comment:
not sure if I can suggest a better name, but simple doesn't convey the behavior...(For tracing, we could use Simple, as spec defined it).
Manual? Basic?

Reply:
Yea, I'm not a fan of `Simple` here either. I think `Manual` conveys behavior best.

I was tempted to consider the terms `Push` vs. `Pull`, but those are terms that primarily describe metric exporters not metric readers... if this is true, then I say let's go with `Manual`.

Reply:
Renamed `Simple` to `Manual`.

Reply:
I don't know if `Manual` is a good name, there are cases (e.g. during dispose) it flushes automatically. My gut feeling is `Basic` > `Simple` > `Manual`. Not a blocking comment though, as I'm always struggling and bad at naming 🤷‍♂️

Reply:
> I don't know if Manual is a good name, there are cases (e.g. during dispose) it flushes automatically

Well..the dispose is a manual action, as user has to explicitly call it (.Dispose, or use using statement/block to let c# do it)

I'll merge to unblock progress, and will do a quick vote in todays SIG meeting:
1. Any alternate name
2. Or pick among the existing Basic, Simple, Manual

Reply:
OnDemand?

Reply:
How about AdHoc?

Reply:
Disordered 🤣